templates/common/infra/bicep/core/database/mysql/mysql-database.bicep

```diff
@@ -0,0 +1,18 @@
+param name string = ''
+param mysqlServerName string = ''
+
+resource database 'Microsoft.DBforMySQL/flexibleServers/databases@2021-05-01' = {
+  parent: mysqlServer
+  name: name
+  properties: {
+    charset: 'utf8'
+    collation: 'utf8_general_ci'
+  }
+}
+
+resource mysqlServer 'Microsoft.DBforMySQL/flexibleServers@2021-05-01' existing = {
+  name: mysqlServerName
+}
+
+output jdbcUrl string = 'jdbc:mysql://${mysqlServer.properties.fullyQualifiedDomainName}:3306/${name}?useSSL=true&requireSSL=false'
```

Comment:
Can we get a better name for this variable?  We typically use "endpoint" and more verbosity to reduce confusion.